<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<!-- Launch point cloud labeling node -->
	<node name="pcl_labeling_node" pkg="pcl_labeling" type="pcl_labeling_node" output="screen"/>
</launch>